src/main/java/com/amazon/sqs/javamessaging/util/ExponentialBackoffStrategy.java

```diff
@@ -33,7 +33,7 @@ public ExponentialBackoffStrategy(long delayInterval, long initialDelay, long ma
     /**
      * Returns the delay before the next attempt.
      * 
-     * @param retriesAttempted
+     * @param retriesAttempted number of retries attempted
```

Comment:
This parameter description doesn't really convey more than what the variable conveys. In such cases, I think we should just skip the description.

Reply:
agree. will change it